orchestration/dap_orchestration/solids.py

```diff
@@ -82,7 +103,8 @@ def env_extract_records(config: dict[str, str]) -> dict[str, str]:
         "output_prefix": String,
         "target_class": String,
         "scala_project": String
-    }
+    },
+    input_defs=[InputDefinition("start", Nothing)]
```

Comment:
Is this the piece that allows us to pass the extract functions to the transform ones in the pipelines file?

Reply:
Yes, exactly. This is aka a[ "nothing dependency"](https://docs.dagster.io/concepts/solids-pipelines/pipelines#order-based-dependencies-nothing-dependencies), purely there to indicate ordering to dagster.